### PR TITLE
Create netex_utils module with 'get_value_in_keylist'

### PR DESCRIPTION
**Pull request by woshilapin**
Starting to move some `utils` out of hellogo fares in a separate module so we can use them in Netex IDF. This PR is only a start (to keep the PR small), I'm also working on getting more in this `utils` module (like stuff about the frames).

### CODE REVIEW

File: src/hellogo_fares/read.rs [hunk 1]
```diff
@@ -169,7 +169,7 @@ fn get_line_id(fare_frame: &Element, service_frame: &Element) -> Result<String>
                     .map(|id| id == line_ref)
                     .unwrap_or(false)
             })
-            .map(|line| utils::get_value_in_keylist(line, "KV1PlanningLijnNummer"))
+            .map(|line| netex_utils::get_value_in_keylist(line, "KV1PlanningLijnNummer"))
```
Comment (ArnaudOggy) [inline src/hellogo_fares/read.rs]: It's really too strange to have Netex and KV1 mixed up. 
It should better with a generic name (like TransModel or something else).

Comment (woshilapin) [reply src/hellogo_fares/read.rs]: HelloGo Fares is Netex. KV1 is for the Public Transport data but the fares in HelloGo is using Netex format.

**Comment by woshilapin** [comment]
@datanel The one function in a module is only temporary. I'm already working on moving some other stuff in it (as mentioned in my PR description).

File: src/hellogo_fares/read.rs (hunk 1 above)
Comment (ArnaudOggy) [reply src/hellogo_fares/read.rs]: Okay. My memory is lacking on this Monday morning :laughing:

**Pull request merged**